docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: '3.8'
+
+# Ports
+# Builder: 3000
+# Explorer: 3001
+# Manager: 3002
+# Pinata: 3003
+# Indexer: 3004
+# Dev chain 1: 3005
+# Subgraph dev chain 1: 3006
+# Dev chain 2: 3007
+# Subgraph dev chain 2: 3008
+
+services:
+  pinata:
+    image: ghcr.io/gitcoinco/pina:init
+    ports:
+      - '127.0.0.1:3003:8000'
+
+  localchain-1:
+    image: ghcr.io/gitcoinco/allo-contracts-v1:pr-1
+    ports:
+      - '127.0.0.1:3005:8545'
+    environment:
+      - DEV_CHAIN_ID=313371
+
+  localchain-1-init:
+    image: ghcr.io/gitcoinco/allo-contracts-v1:pr-1
```

Comment:
I guess the tag `pr-1` will change once this is merged to master. Should this be changed here already?

Reply:
`main` doesn't exists anymore, we can merge this and update it later or review and merge the other one before this one: https://github.com/gitcoinco/allo-contracts-v1/pull/1